app/src/views/GoogleLogin.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+import { AuthSession } from 'expo';
+
+const Google_APP_ID = '847385043058-1v26bjo0eahcc7sal4f86s7ed04mtjro.apps.googleusercontent.com';
+
+export default class App extends React.Component {
```

Comment:
Can we rename this class as we already have an App class in the App.js file

Reply:
yeah I just renamed it to Login